modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala

```diff
@@ -185,12 +186,19 @@ object NewPullRequestData {
       )
     }
 
-  def semVerLabel(update: Update): Option[String] =
+  def earlySemVerLabel(update: Update): Option[String] =
     for {
-      curr <- SemVer.parse(update.currentVersion)
-      next <- SemVer.parse(update.nextVersion)
-      change <- SemVer.getChange(curr, next)
-    } yield s"semver-${change.render}"
+      curr <- EarlySemVer.parse(update.currentVersion)
+      next <- EarlySemVer.parse(update.nextVersion)
+      change <- EarlySemVer.getChange(curr, next)
+    } yield s"early-semver-${change.render}"
+
+  def semVerSpecLabel(update: Update): Option[String] =
+    for {
+      curr <- SemVerSpec.parse(update.currentVersion)
+      next <- SemVerSpec.parse(update.nextVersion)
+      change <- SemVerSpec.getChange(curr, next)
+    } yield s"semver-spec-${change.render}"
```

Comment:
Note) Changing `semver-${change.render}` to `semver-spec-${change.render}` seems breaking changes for existing automation.

Reply:
Yep, that's true. But I believe the change is worth it, especially for the sake of automation.